index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main>
+        <nav class="nav-bar">
+            <ul>
+                <li><a href="#"> <img class="apple-img" src="images/apple.png"> </a></li>
+                <li><a href="#"> Store </a></li>
+                <li><a href="#"> Mac </a></li>
+                <li><a href="#"> iPod </a></li>
+                <li><a href="#"> iPhone </a></li>
+                <li><a href="#"> iPad </a></li>
+                <li><a href="#"> iTunes </a></li>
+                <li><a href="#"> Support </a></li>
+                <li><img class="search-icon" src="images/searchicon.png"> <input type="text"></li>
+            </ul>
+        </nav>
+        <section class="main-header">
+            <a href="#">
+                <div class="background-image hero">
+                    <img class="img-title" src="images/your_verse_title_small.png"> 
+                    <img class="img-text" src="images/your_verse_subhead_small.png">
+                    <div class="box-image">
+                        <img src="images/cta_your_verse_27pt.png">
+                    </div>
+                </div>
+            </a>
+        </section>
+        <section class="info-list">
+            <a href="#"><div class="background-image info" id="iphone5s"></div></a>
+            <a href="#"><div class="background-image info" id="iphone5c"></div></a>
+            <a href="#"><div class="background-image info" id="30_years"></div></a>
+            <a href="#"><div class="background-image info" id="supplier_responsibility"></div></a>
+        </section>
```

Comment:
This section tag should go in the footer and pass the images and background images

Reply:
Now the section is in the footer.
Thank you for all.